openedx/core/djangoapps/extensions_lite/__init__.py

```diff
@@ -0,0 +1,7 @@
+"""
+The tiny subset of the django-extensions package that we were actually using.
+
+Replicated here because django-extensions was proving difficult to keep up to
+date thanks to its large surface area for bugs and Django version
+incompatibility.
+"""
```

Comment:
That might work; I hadn't spotted it because there are no management commands currently in it.  I think @macdiesel recently had some import conflict problems with that one, but it shouldn't be a problem for loading management commands.